float32 goal_x
float32 goal_y
---
int8 pushback
